src/api_v2/node.md

```diff
@@ -595,6 +595,70 @@ HTTP/1.1 200 OK
 
       */
 
+== [POST] api/nodes/{id}
+
+    /**
+    @api {post} /api/nodes/{id} 5. Update Node settings
+    @apiVersion 8.0.0
+    @apiName updateNode
+    @apiGroup Nodes
+
+    @apiUse nodeId
+    @apiParam (Multi) {Property}       properties A property of the Node defined by its name and its value. Value are non-empty UTF-8 strings. Setting a key to the empty string removes it from the list of properties for that node.
+    @apiParam {PolicyMode}       policy In which mode the node will apply its configuration policy. You can choose between "audit" and "enfoce" modes, or "default" to use the global mode
+
+    @apiDescription This API allows to update the node settings.
+
+    @apiExample Initial content
+# Given the "data.json" JSON file with content:
+{ "properties": [
+  { "name": "env_type"    , "value": "production" },
+  { "name": "shell"       , "value": "/bin/sh" },
+  { "name": "utf-8 poetry", "value": "ᚠᛇᚻ᛫ᛒᛦᚦ᛫ᚠᚱᚩᚠᚢᚱ᛫ᚠᛁᚱᚪ᛫ᚷᛖᚻᚹᛦᛚᚳᚢᛗ" }
+] 
+, "policyMode" : "audit"
```

Comment:
you should start with enforce to let the user understand that "policyMode=default" actually changed something in the example call.